packages/@expo/package-manager/__mocks__/node:fs.ts

```diff
@@ -1,2 +0,0 @@
-import { fs } from 'memfs';
```

Comment:
This is required for mocking imports to `node:fs`, you'll need an alternative mock if you want to delete the file.

Reply:
Do you know what test is using this?  -- I tried running yarn test in the @expo/package-manager package, and didn't hi an issue with this file deleted.

Lots of the existing tests are manually doing this mock, so it maybe that it is not needed.  Ex: https://github.com/expo/expo/blob/0b69e7a983625dd42e5a5b4447e5f94f327441ab/packages/%40expo/package-manager/src/utils/__tests__/nodeManagers-test.ts#L19

Reply:
Yeah perhaps this module isn't using it and it's just for consistency. Weird that you'd only have a problem with this particular package's mock though.

Reply:
The issue is the file name.  This is the only mock with ':' in the file name.  And ':'s are not an allowed character within a filename on windows.